api/lanes.py

```diff
@@ -906,26 +917,26 @@ def url_arguments(self):
         return self.ROUTE, kwargs
 
     def featured_works(self, _db):
-        qu = self.works(_db)
-
-        # Aliasing Edition here allows this query to function
-        # regardless of work_model and existing joins.
-        work_edition = aliased(Edition)
-        qu = qu.join(work_edition).order_by(work_edition.series_position, work_edition.title)
-        target_size = self.get_library(_db).featured_lane_size
-        qu = qu.limit(target_size)
```

Comment:
The motivation behind FeaturedQueryFacets is getting rid of this code, which is already present, more or less, in Facets. This code broke when the rules about DISTINCT changed.